src/assets/like-button/question-btn.svg

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg version="1.1" viewBox="0 0 16 16" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
+<svg version="1.1" viewBox="0 0 16 16" fill="#9B9B9B" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
```

Comment:
So that we can control color of the SVG by setting text color
```suggestion
<svg version="1.1" viewBox="0 0 16 16" fill="currentColor" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
```